Snowflake.Data/Core/Session/SFSessionProperty.cs

```diff
@@ -215,7 +215,7 @@ internal static SFSessionProperties ParseConnectionString(string connectionStrin
                 }
                 catch (ArgumentException e)

```

Comment:
Can you remove variable `e` as it generates warnings.

Reply:
Fixed